code/modules/mob/living/carbon/human/species_types/polysmorphs.dm

```diff
@@ -94,13 +94,55 @@
 /datum/species/polysmorph/create_pref_unique_perks()
 	var/list/to_add = list()
 
-	// TODO
-
-	return to_add
-
-/datum/species/polysmorph/create_pref_biotypes_perks()
-	var/list/to_add = list()
-
-	// TODO
+	to_add += list(
+		list(
+			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
+			SPECIES_PERK_ICON = "running",
+			SPECIES_PERK_NAME = "Predator Genes",
+			SPECIES_PERK_DESC = "Polysmorphs keep a fraction of the agility found in their xenomorph ancestors. \
+								Their movement speed and crawl speed are slightly faster than most races.",
+		),
+		list(
+			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
+			SPECIES_PERK_ICON = "low-vision",
+			SPECIES_PERK_NAME = "Darkvision",
+			SPECIES_PERK_DESC = "Polysmorphs have an advanced set of eyes protected inside their domed head. \
+								These eyes provide a degree of night-vision, letting them see better in the dark than most.",
```

Comment:
Nah, now its infrared vision, not night vision
Try to make it clear that its NOT thermal vision